CMakeLists.txt

```diff
@@ -9,7 +9,8 @@ include(GNUInstallDirs)
 set(SSG_CONTENT_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/xml/scap/ssg/content")
 set(SSG_GUIDE_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/guides")
 set(SSG_TABLE_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/tables")
-set(SSG_ROLE_INSTALL_DIR "${CMAKE_INSTALL_DOCDIR}/roles")
+set(SSG_ROLE_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/scap-security-guide/roles")
+set(SSG_BASH_INSTALL_DIR "${CMAKE_INSTALL_DATADIR}/scap-security-guide/bash")
```

Comment:
We should do better with the variable names.

SSG_ANSIBLE_ROLES_INSTALL_DIR
SSG_BASH_ROLES_INSTALL_DIR

I also suggest installign ansible roles to `scap-security-guide/ansible`. Even for an Ansible user this will be easier to discover.